src/api/institutionApi.ts

```diff
@@ -75,3 +75,21 @@ export async function getInstitutionName(
     cachedInstitutionResult: cachedInstitutionResult,
   };
 }
+
+export async function getInstitutionNameNoCache(
```

Comment:
Jeg gav opp å finne ut hvordan den gamle koden hang sammen  og lagde en ny api-metode isteded. 

Kanskje vi kunne ha brukt et cache-adapter på axios for å forenkle cache-problematikken ?